site/en/docs/devtools/progressive-web-apps/index.md

```diff
@@ -35,54 +40,69 @@ panel guides][2].
 
-You can test out this feature with the [Google I/O 2016 progressive web app][5], which has proper
-support for Add to Homescreen. Clicking on **Add to Homescreen** while the app is open prompts
-Chrome to display the "add this site to your shelf" banner, which is the desktop equivalent of the
-"add to homescreen" banner for mobile devices.
+   {% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/jiBO2oDEU5TYKKBNfuMc.png", alt="The Install button.", width="800", height="577" %}
 
-{% Img src="image/admin/dcHC145P0QVJTCapLWFd.png", alt="add to desktop shelf", width="800", height="611" %}
+1. Follow the on-screen instructions.
 
-**Tip**: Keep the **Console** drawer open while simulating Add to Homescreen events. The Console
-tells you if your manifest has any issues and logs other information about the Add to Homescreen
+{% Aside 'gotchas' %}
+Keep the **Console** drawer open when you trigger installation. The **Console**
+tells you if your manifest has any issues and logs other information about the installation
 lifecycle.
+{% endAside %}
 
-The **Add to Homescreen** feature cannot yet simulate the workflow for mobile devices. Notice how
-the "add to shelf" prompt was triggered in the screenshot above, even though DevTools is in Device
-Mode. However, if you can successfully add your app to your desktop shelf, then it'll work for
+The **Install app** feature cannot simulate the workflow for mobile devices. Notice how
+the desktop Chrome browser displays the installation button in the address bar, even though DevTools is in [Device Mode](/docs/devtools/device-mode/).
+However, if you can successfully add your app to your desktop, then it'll work for
 mobile, too.
 
 If you want to test out the genuine mobile experience, you can connect a real mobile device to
-DevTools via \[remote debugging\][remote debugging][6], and then click the **Add to Homescreen**
-button (on DevTools) to trigger the "add to homescreen" prompt on the connected mobile device.
+DevTools via [remote debugging][6]. To trigger the installation on the connected mobile device, open the {% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/N5Lkpdwpaz4YqRGFr2Ks.svg", alt="Three-dot menu.", width="22", height="22" %} three-dot menu and click {% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/YwNSnnzuZ3dJwtVhcC4I.svg", alt="Install app.", width="24", height="24" %} **Install app**.
+
+### Test URL protocol handler registration {: #test-protocol-handler }
+
+The registration of such a handler lets installed PWAs handle links that use the URL protocol.
+To learn how to create a handler, see [URL protocol handler registration for PWAs](https://web.dev/url-protocol-handler/).
+
+To test your handler:
+
+1. [Open DevTools](/docs/devtools/open/) on the landing page of your PWA. For example, check out this [demo page](https://protocol-handler.glitch.me/).
```

Comment:
Missing a step. To test the protocol handler (on successful navigation), one need to first install the app and reload and register.